sonar-orchestrator/src/main/java/com/sonar/orchestrator/Orchestrator.java

```diff
@@ -273,7 +273,9 @@ private BuildResult executeBuildInternal(Build<?> build, boolean quietly, boolea
 
   /**
    * Reset inspection measures and some other data (manual rules, etc.)
+   * @deprecated in 3.16. Tests should use web services to clean-up state.
    */
+  @Deprecated
   public void resetData() {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Do not forget to remove this deprecated code someday. ([squid:S1133](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.orchestrator%3Aorchestrator-parent&branch=feature%2Fsb%2FORCH-409%2Fartifactory&issues=AWG-CB_g_Zsgh39as5A4&open=AWG-CB_g_Zsgh39as5A4)

[ISSUE_KEY]: AWG-CB_g_Zsgh39as5A4